modules/compbox/templates/nginx.conf.erb

```diff
@@ -33,17 +33,18 @@ server {
     }
 
     location = /stream {
+        return 307 /stream/;
```

Comment:
In the past we've seen hard-to-diagnose issues with redirecting stream connections (see #13 and https://github.com/PeterJCLaw/srcomp-puppet/commit/9ede7dc0e1d08587d2f1fc1afac4bf096a8bb95b). If possible I think we should maintain the existing url.

Reply:
The other changes here direct the stream directly to `/stream/`. I've added the CORS header to match the stream header and tested it works with both the javascript EventStream and python library.